ipwb/util.py

```diff
@@ -42,33 +44,16 @@
 dtPattern = re.compile(r"^(\d{4})(\d{2})?(\d{2})?(\d{2})?(\d{2})?(\d{2})?$")
 
 
-def getMultiAddress(hostAndPort=IPFSAPI_ADDRESS):
-    try:
-        (host, port) = hostAndPort.split(':')
-    except ValueError as err:
-        # Naive check to determine if multiaddr passed in
-        if hostAndPort[0] == '/' and hostAndPort.count('/'):
-            return hostAndPort
-        else:
-            print('Error reading IPFSAPI address')
-            sys.exit()
-    try:
-        socket.inet_aton(host)
-        return f"/ip4/{host}/tcp/{port}/http"
-    except socket.error:
-        return f"/dns/{host}/tcp/{port}/http"
-
-
-def isDaemonAlive(hostAndPort=IPFSAPI_ADDRESS):
+def isDaemonAlive(daemonMultiaddr=IPFSAPI_MUTLIADDRESS):
     """Ensure that the IPFS daemon is running via HTTP before proceeding"""
-    client = ipfsapi.Client(getMultiAddress(hostAndPort))
+    client = ipfsapi.Client(daemonMultiaddr)
```

Comment:
I would wrap this client creation line in a try-except block, because this is where any multi-addr formatting issues would be validated. This means, if the user supplies an invalid multi-addr, an appropriate exception will be thrown here.